### PR TITLE
QuestionsAnswers.tsx added using shadcn/ui library 

### PR DESCRIPTION
**Pull request by develad**
localhost_3000_maintainers(iPad Air)1

### CODE REVIEW

**Comment by develad** [comment]
> Nice PR, what are the benefits of using shadcn and radix/tailwind animate if there is already an animation lib in the dependencies? I might miss something

Nice comment. There isn't any special benefit by using those libraries as far as i know. You are more than welcome to implement an accordion by your own with the existing libraries 😃

**Comment by adirkandel** [comment]
@develad You did an amazing job buddy, tnx!

I see that you installed radix accordion so, did you use radix or shadcn?

Also, i join to @rotembarsela comment, we should use one library to a specific use, in this case animation from a several cases - maintenance, dependencies, conventions, etc.
As a group of contributors we should strive to have a coherent project so it will be easy for us to understand the code.

As a maintainer i must keep these guidelines and reduce questions like which library should i use for animation so it will be easier for others to understand the project.

File: package.json [hunk 1]
```diff
@@ -17,18 +17,23 @@
     ]
   },
   "dependencies": {
+    "@radix-ui/react-accordion": "^1.1.2",
     "autoprefixer": "10.4.14",
+    "class-variance-authority": "^0.7.0",
```
Comment (adirkandel) [inline package.json:22]: Do we need this library? because i can't where we use it

File: tailwind.config.js [hunk 2]
```diff
@@ -10,7 +10,28 @@ module.exports = {
     './app/**/*.{js,ts,jsx,tsx,mdx}',
   ],
   theme: {
+    container: {
```
Comment (adirkandel) [inline tailwind.config.js:13]: we already have `container` prop here, can you merge between them?

**Comment by adirkandel** [comment]
@develad Can you please resolve conflicts? also, for some reason i can't see the Hero section you made in the Maintainers page.

**Pull request closed without merging**